site/app/site/scripts/views/form.js

```diff
@@ -29,6 +29,10 @@ define(["jquery", "underscore", "backbone", "models/account", "validation"],
                         window.location = "/site/maintenance";
                     }
                 });
+
+                // make first get request which fetches csrf token
+                Account.getUserSettings();
```

Comment:
Account.getUserSettings() method returns success in both cases (if user logged in it returns his settings, in other case it returns an empty object).  What will happen if you get unexpected error in response?